examples/todomvc/components/itemview.go

```diff
@@ -52,25 +45,25 @@ func (p *ItemView) onToggleCompleted(event *vecty.Event) {
 func (p *ItemView) onStartEdit(event *vecty.Event) {
 	p.editing = true
 	p.editTitle = p.Item.Title
-	p.ReconcileBody()
-	p.input.Node().Call("focus")
+	vecty.Rerender(p)
```

Comment:
Instead of components calling `(*MyComponent).Composite.ReconcileBody()`, they just call `vecty.Rerender(myComponent)`.

While `Rerender` uses the component's `vecty.Core` embedded struct internally, the overall end-user perception is much more straightforward. Also `Rerender` sounds simpler / more understandable than `ReconcileBody` (easy to confused with the DOM `<body>` tag).